lib/collection/property.js

```diff
@@ -170,6 +170,15 @@ _.extend(Property, /** @lends Property */ {
      * @returns {String}
      */
     replaceSubstitutions: function (str, variables) {
+        var loopCounter = -1,
+            resolvedStr = '',
```

Comment:
You don't need to initialize it to a blank string, just declaring the var should be enough